ovs/lib/scheduledtask.py

```diff
@@ -225,7 +226,11 @@ def gather_scrub_work():
                     logger.info('Gather Scrub - Storage Router {0:<15} has SCRUB partition at {1}'.format(storage_driver.storagerouter.ip, partition.path))
                     if storage_driver.storagerouter not in scrub_locations:
                         try:
-                            _ = SSHClient(storage_driver.storagerouter)
+                            sshclient = SSHClient(storage_driver.storagerouter)
+                            # Use ServiceManager(sshclient) to make sure ovs-workers are actually running
+                            if ServiceManager.get_service_status('workers', sshclient) is False:
+                                logger.warning('Gather Scrub - Storage Router {0:<15} - workers are not running'.format(storage_driver.storagerouter.ip))
+                            #else:
```

Comment:
Why add the storagerouter to the list of ```scrub_locations``` even when the workers are not running?   And if this is what we want, please remove the ```#else:```